besu/src/main/java/org/hyperledger/besu/cli/options/stable/NodePrivateKeyFileOption.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
nit: new files should have * Copyright contributors to Hyperledger Besu  https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy